index.js

```diff
@@ -63,8 +63,8 @@ var codes = exports.code = exports.codes = {
   'down': 40,
   'insert': 45,
   'delete': 46,
-  'command': 91,
```

Comment:
Should probably alias left command and command, in case someone currently relying on current 'command' result.